### DISCUSSION TITLE
custom domain name

### DISCUSSION THREAD

**Discussion by Habiba-Mohammad** [community/community#56415] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

the custom domain name gives me this message for 3 days " DNS is in progress" how can I fix it?

**Comment by hrishikesh-kocharekar-dev** [comment]
If you're seeing the message "DNS is in progress" for your custom domain name for several days, it typically means that the DNS changes you made have not propagated fully across the internet. DNS propagation can take anywhere from a few hours to up to 48 hours or more, depending on various factors such as your DNS provider and the TTL (Time to Live) settings of your DNS records.

you can 
Verify DNS configuration: Double-check your DNS configuration to ensure that you have correctly set up the necessary DNS records for your custom domain. Ensure that the records point to the correct IP address or hostname of your server or hosting provider.

Reduce TTL values: Lower the TTL (Time to Live) values for your DNS records. TTL determines how long DNS resolvers and caches will hold onto old DNS information before checking for updates. By reducing the TTL, you can help propagate changes more quickly. However, note that changes to TTL may take some time to take effect.

Flush DNS cache: Clear the DNS cache on your local machine and your network. You can do this by running the appropriate commands based on your operating system. For example, on Windows, you can open the command prompt and run `ipconfig /flushdns`.

Check with your DNS provider: Reach out to your DNS provider and inquire about any potential issues or delays they might be experiencing. They can provide more information and help troubleshoot the problem.

Wait patiently: In some cases, DNS propagation can simply take longer than expected due to various factors. It's important to be patient and wait for the changes to propagate fully across the internet. Be aware that different regions and ISPs may have varying propagation times.
